api/alert_rules.go

```diff
@@ -40,15 +40,22 @@ var AlertRuleCategories = []string{"Anomaly", "Policy", "Composite"}
 // Valid inputs for AlertRule SubCategories property
 var AlertRuleSubCategories = []string{
```

Comment:
Pity these are not enum.  Lines 153+ could refer to a type instead of repeating the strings.

Reply:
Yes. The strings will be used in terraform to enforce valid values. Will leave it to keep it simple.